zetaclient/chains/solana/contract/contract.go

```diff
@@ -0,0 +1,38 @@
+package contract
+
+const (
+	// GatewayProgramID is the program ID of the Solana gateway program
+	GatewayProgramID = "94U5AHQMKkV5txNJ17QPXWoh474PheGou6cNP2FEuL1d"
```

Comment:
To be replaced with chain params right? We should create TODO + issue link if so

Reply:
This will be the only place that defines the gateway program address constant.

Reply:
https://github.com/zeta-chain/node/pull/2465/commits/eb814a91d89bee5be48174081af7338f76cb197a